Tests/SwiftPrometheusTests/PrometheusMetricsTests.swift

```diff
@@ -142,7 +142,32 @@ final class PrometheusMetricsTests: XCTestCase {
         my_summary{quantile="0.99", myValue="labels"} 123.0
         my_summary{quantile="0.999", myValue="labels"} 123.0
         my_summary_count{myValue="labels"} 1
-        my_summary_sum{myValue="labels"} 123
+        my_summary_sum{myValue="labels"} 123.0
+        """)
+    }
+    
+    func testSummaryWithPreferredDisplayUnit() {
+        let summary = Timer(label: "my_summary", preferredDisplayUnit: .seconds)
+        
+        summary.recordSeconds(1)
+        summary.recordSeconds(2)
+        summary.recordSeconds(4)
```

Comment:
```suggestion
        summary.recordNanoseconds(4 * 1_000_000_000)
```

I think it'd be nice to also exercise other units to make sure we don't regress in the future.